nodeup/pkg/model/kubecontrollermanager.go

```diff
@@ -188,6 +189,14 @@ func (b *KubeControllerManagerBuilder) buildPod() (*v1.Pod, error) {
 		},
 	}
 
+	if os.Getenv("HTTP_PROXY") != "" || os.Getenv("HTTPS_PROXY") != "" {
```

Comment:
Is there a project convention for helper functions that span `nodeup`, `kops`, and `protokube`?